packages/faceted-search/src/components/ClearBasicSearch/ClearBasicSearch.component.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ActionButton from '@talend/react-components/lib/Actions/ActionButton';
+import { getTheme } from '@talend/react-components/lib/theme';
+import { USAGE_TRACKING_TAGS } from '../../constants';
+import cssModule from './ClearBasicSearch.scss';
+
+const theme = getTheme(cssModule);
+
+export default function ClearBasicButton({ t, onClick, isDisabled }) {
```

Comment:
This component is just a styled/configured dumb button. I think you can inline it in the BasicSearch component.
So we can get rid of this file and the tests (it should be tested in the BasicSearch component for the click)

Reply:
Done